frontend/src/component/Account/UserSpacesConfig.jsx

```diff
@@ -66,43 +56,44 @@ export const UserSpacesConfig = (props) => {
       { entityType: TLM_ET.SHAREDSPACE_MEMBER, coreEntityType: TLM_CET.MODIFIED, handler: handleMemberModified },
       { entityType: TLM_ET.SHAREDSPACE_MEMBER, coreEntityType: TLM_CET.DELETED, handler: handleMemberDeleted }
     ])
-  }, [spaceList])
+  })
 
   useEffect(() => {
     const filteredListWithMember = []
+    if (spaceList !== undefined) {
+      spaceList.forEach(space => {
+        const member = space.memberList.find(u => u.id === props.userToEditId)
+        if (space.memberList.length > 0 && member) {
+          filteredListWithMember.push({ ...space, member })
+        }
+      })
 
-    spaceList.forEach(space => {
-      const member = space.memberList.find(u => u.id === props.userToEditId)
-      if (space.memberList.length > 0 && member) {
-        filteredListWithMember.push({ ...space, member })
-      }
-    })
-
-    const sortedList = sortListBy(
-      filteredListWithMember,
-      selectedSortCriterion,
-      sortOrder,
-      props.user.lang
-    )
-
-    const filteredSpaceList = filterSpaceList(sortedList)
-
-    const entryList = filteredSpaceList.map(space => {
-      return (
-        <UserSpacesConfigLine
-          space={space}
-          key={space.id}
-          onChangeEmailNotificationType={
-            emailNotificationType => props.onChangeEmailNotificationType(space.id, emailNotificationType)
-          }
-          onLeaveSpace={handleLeaveSpace}
-          admin={props.admin}
-          system={props.system}
-          onlyManager={onlyManager(props.userToEditId, space.member, space.memberList)}
-        />
+      const sortedList = sortListBy(
+        filteredListWithMember,
+        selectedSortCriterion,
+        sortOrder,
+        props.user.lang
       )
-    })
-    setEntries(entryList)
+
+      const filteredSpaceList = filterSpaceList(sortedList)
+
+      const entryList = filteredSpaceList.map(space => {
+        return (
+          <UserSpacesConfigLine
+            space={space}
+            key={space.id}
+            onChangeEmailNotificationType={
+              emailNotificationType => props.onChangeEmailNotificationType(space.id, emailNotificationType)
+            }
+            onLeaveSpace={handleLeaveSpace}
+            admin={props.admin}
+            system={props.system}
+            onlyManager={onlyManager(props.userToEditId, space.member, space.memberList)}
+          />
+        )
+      })
+      setEntries(entryList)
```

Comment:
with the fix for registering the TLM once at component mount, you need to update this setter to use the proper way of updating a value from the previous one

Reply:
the value of entries isn't updating from the previous one